src/analysis/special_functions/polynomials.lean

```diff
@@ -147,4 +156,17 @@ begin
     exact tendsto_abs_at_bot_at_top.comp (P.div_tendsto_at_bot_of_degree_gt Q hdeg hQ h.le) }
 end
 
+theorem is_O_of_degree_le (h : P.degree ≤ Q.degree) :
+  is_O (λ x, eval x P) (λ x, eval x Q) filter.at_top :=
```

Comment:
It would be nice to strengthen this to an `iff` statement.

Reply:
The reverse direction of this seems non-trivial to prove, so I'll plan on doing this in a follow-up PR.